app/services/notifications/milestone.rb

```diff
@@ -0,0 +1,5 @@
+module Notifications
+  module Milestone
+    ARTICLE_FINAL_PUBLICATION_TIME_FOR_MILESTONE = Time.zone.local(2019, 2, 25)
```

Comment:
The constant is lonely here :) I suggest moving it to the class as well.
There is a space for further refactoring, but it's not a topic of the pr.

Reply:
Yes, it was a bit alone, I have moved it with the send class.